db/migrate/20220504125616_add_show_on_homepage_flag_and_badge_uri_to_courses.rb

```diff
@@ -0,0 +1,6 @@
+class AddShowOnHomepageFlagAndBadgeUriToCourses < ActiveRecord::Migration[6.1]
+  def change
+    add_column :courses, :show_on_homepage, :boolean, default: false, null: false
+    add_column :courses, :badge_uri, :string
```

Comment:
Now that we have this badge_uri field available to us, what are your thoughts on going all-in with it by adding a null constraint and setting all the course badges in the seeds?